BuildingSync.xsd

```diff
@@ -3382,6 +3417,42 @@
         </xs:complexType>
       </xs:element>
       <xs:element ref="auc:UserDefinedFields" minOccurs="0"/>
+      <xs:element minOccurs="0" name="AnnualAverageGHGEmissions">
```

Comment:
The new "GHG" elements should be declared before the `auc:UserDefinedFields` element.

Reply:
@markborkum, good catch! I just switched the order.